lib/jxl/enc_adaptive_quantization.cc

```diff
@@ -329,20 +335,23 @@ void FuzzyErosion(const Rect& from_rect, const ImageF& from,
       size_t xp1 = x + kStep < xsize ? x + kStep : x;
       float min0 = row[x];
       float min1 = min0;
-      float min2 = min1;
-      StoreMin3(row[xm1], min0, min1, min2);
-      StoreMin3(row[xp1], min0, min1, min2);
-      StoreMin3(rowt[xm1], min0, min1, min2);
-      StoreMin3(rowt[x], min0, min1, min2);
-      StoreMin3(rowt[xp1], min0, min1, min2);
-      StoreMin3(rowb[xm1], min0, min1, min2);
-      StoreMin3(rowb[x], min0, min1, min2);
-      StoreMin3(rowb[xp1], min0, min1, min2);
-      static const float kMulC = 0.029598804634393225 * 0.25f;
-      static const float kMul0 = 0.561331076516815 * 0.25f;
-      static const float kMul1 = 0.16504828561110252 * 0.25f;
-      static const float kMul2 = 0.2440218332376892 * 0.25f;
-      float v = kMulC * row[x] + kMul0 * min0 + kMul1 * min1 + kMul2 * min2;
+      float min2 = min0;
+      float min3 = min0;
+      StoreMin3(row[xm1], min0, min1, min2, min3);
+      StoreMin3(row[xp1], min0, min1, min2, min3);
+      StoreMin3(rowt[xm1], min0, min1, min2, min3);
+      StoreMin3(rowt[x], min0, min1, min2, min3);
+      StoreMin3(rowt[xp1], min0, min1, min2, min3);
+      StoreMin3(rowb[xm1], min0, min1, min2, min3);
+      StoreMin3(rowb[x], min0, min1, min2, min3);
+      StoreMin3(rowb[xp1], min0, min1, min2, min3);
+      static const float kMulC = 0.05f;
+      static const float kMul0 = 0.05f;
+      static const float kMul1 = 0.05f;
+      static const float kMul2 = 0.05f;
+      static const float kMul3 = 0.05f;
+      float v = kMulC * row[x] + kMul0 * min0 + kMul1 * min1 + kMul2 * min2 +
```

Comment:
What is the intuitive/high-level purpose of this change? Do the `kMul*` constants still need tweaking?

Reply:
just penalize neighbourhoods with more blank area around them even more

neither before or after is a disciplined approach that strictly makes sense

Reply:
with very weak evidence, this seemed better, or at least doesn't hurt with all the other changes